### PR TITLE
Add deploy to org

### PR DESCRIPTION
**Pull request by mgagliardo**
# What? 
Deploy to org with previous approval on CircleCI

# Why?
We don't want @eordano to be doing sysadmin's work anymore.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -91,50 +87,89 @@ jobs:
...
     steps:
       - restore_cache:
           keys:
             - build-{{ .Revision }}
       - run: sudo apt-get -y -qq install awscli
       - run:
-          name: Deploy to .zone if tests pass and branch is Master
-          command: aws s3 sync static s3://explorer.decentraland.zone/ --exclude "branch/*" --acl public-read
+          name: Deploy to S3 under subfolder if tests pass and branch is not master
+          command: aws s3 sync static s3://explorer.decentraland.zone/branch/${CIRCLE_BRANCH} --acl public-read
```
Comment (menduz) [inline .circleci/config.yml:105]: Why this change? currently we are deploying to `/` so we access `index.html`when entering `https://explorer.decentraland.zone` this will require us to enter `https://explorer.decentraland.zone/master`

**Review by menduz — CHANGES_REQUESTED** [review]
Only .zone seems different, the rest is OK. Left a comment in the .zone line

**Review by eordano — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -144,15 +185,43 @@ workflows:
...
+            tags:
+              # Match SemVer N.N.N
+              only: /^(\d+\.)?(\d+\.)?(\*|\d+)$/
+      - deploy-prd:
+          requires:
+            - deploy-stg
+            - hold
+          filters:
+            branches:
+              ignore: /.*/
```
Comment (eordano) [inline .circleci/config.yml:228]: :/

**Review by eordano — CHANGES_REQUESTED** [review]

**Comment by mgagliardo** [comment]
Yay [CircleCI's magic](https://discuss.circleci.com/t/workflow-job-with-tag-filter-being-run-for-every-commit/20762/4)
